### PR TITLE
Styling follow buttons without using class and selected_class

### PR DESCRIPTION
**Pull request by luis-almeida**

### CODE REVIEW

File: styles/_my-activities.scss [hunk 1]
```diff
@@ -109,21 +109,21 @@
...
+        @extend .button;
+        line-height: 40px;
       max-height: 40px;
       padding: 0 20px;
-    }
-
-    .organization-unsubscribe {
-      background-color: $button-color;
-      color: $brand_text_color;
+        &[data-selected="true"] {
```
Comment (VilhelmNielsen) [inline styles/_my-activities.scss]: Are we changing these to `aria-pressed` later?

Comment (luis-almeida) [reply styles/_my-activities.scss]: I thought we were discussing `aria-pressed` for votes only.
These changes are related to the follow button/menu-button. When the state changes the label changes as well so not sure `aria-pressed` is appropriate (?) also due to the menu behaviour.

Comment (VilhelmNielsen) [reply styles/_my-activities.scss]: Ah yes, sorry, I'm mixing things up here. Excuse my fried egg brain 🍳

Comment (luis-almeida) [reply styles/_my-activities.scss]: No worries! Same here 🍳

**Pull request merged**